pontoon/base/models.py

```diff
@@ -1335,6 +1378,13 @@ def is_asymmetric(self):
     def __unicode__(self):
         return '%s: %s' % (self.project.name, self.path)
 
+    def save(self, *args, **kwargs):
+        super(Resource, self).save(*args, **kwargs)
+
+        if self.deadline < self.project.deadline:
```

Comment:
What will happen if there will be a couple of resouces with different deadlines:

* resource1 - 2017-02-01
* resource2 - 2017-01-10
I think that current implementation will choose 2017-01-10 as a deadline for the whole project.